Corzbank.Web/ClientApp/src/app/header/header.component.scss

```diff
@@ -0,0 +1,68 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+.navbar{
+    // background-color: green;
+    height: 100px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 25px;
+    letter-spacing: 1px;
+    width: 80%;
+    margin: 10px 0;
+}
+.left-block , .right-block{
+    display: flex;
+    justify-content: space-between;
+    width: 24%;
+    align-items: baseline;
+    align-items: center;
```

Comment:
all styles below overwrite styles from top, that means line 22 overwrite line 21, remove from 21

Reply:
resolved